app/qml/components/ProjectDelegateItem.qml

```diff
@@ -73,6 +73,12 @@ Rectangle {
     return ""
   }
 
+  function getIconColor() {
```

Comment:
best to not use function, just copy paste it to those 2 lines. function does not work with binding events